src/pages/WorkloadList/FiltersAndSorts.ts

```diff
@@ -353,17 +353,6 @@ export const sortWorkloadsItems = (
   unsorted: WorkloadListItem[],
   sortField: SortField<WorkloadListItem>,
   isAscending: boolean
-): Promise<WorkloadListItem[]> => {
-  if (sortField.title === 'Health') {
-    // In the case of health sorting, we may not have all health promises ready yet
-    // So we need to get them all before actually sorting
-    const allHealthPromises: Promise<WithWorkloadHealth<WorkloadListItem>>[] = unsorted.map(item => {
-      return item.healthPromise.then((health): WithWorkloadHealth<WorkloadListItem> => ({ ...item, health }));
-    });
-    return Promise.all(allHealthPromises).then(arr => {
-      return arr.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
-    });
-  }
-  const sorted = unsorted.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
-  return Promise.resolve(sorted);
+): WorkloadListItem[] => {
```

Comment:
Same comment on this file. I think in line 322 the `Promise<WorkloadListItem[]>` return type can be removed, since the `filterBy` function will no longer return that type.